styles.css

```diff
@@ -1,3 +1,793 @@
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 30px;
+    color: #FFFFFF;
+}
+
+.filter-item-selected {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+    color: #F7F7F7;
+}
+
+.contdown {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+}
+
+.countdown-number {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 44px;
+    font-weight: 400;
+    line-height: 56px;
+}
+
+.countdown-unit {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    color: #F6F6F6;
+}
+
+.details-description,
+.caption-description,
+.article-text,
+.reply-sub {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 30px;
+    color: #C2C3C5;
+}
+
+.info-category {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+    color: #C2C3C5;
+}
+
+.info-detail {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+}
+
+.caption-date,
+.caption-author {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #888B8E;
+}
+
+.article-tag {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+}
+
+.blog-details>.article-author,
+.article-date {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 30px;
+    color: #888B8E;
+}
+
+.quote {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 30px;
+}
+
+.rating {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    color: #BBBABE;
+}
+
+.blog-details>.article {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 30px;
+    font-weight: 400;
+    line-height: 42px;
+}
+
+.tag-item a,
+.sidebar-article p,
+.sidebar-article time,
+.tags-list li a {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+}
+
+.article-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 22px;
+    color: #888B8E;
+}
+
+.article-figure .sub-text,
+.sidebar-article>h3 {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 28px;
+    color: #fff;
+}
+
+.categories-link, .filter-item {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    color: #C2C3C5;
+
+}
+
+h2[class~="sidebar"] {
```

Comment:
Why did you decide to use an attribute selector here instead of a class selector?

Attribute selectors (e.g., h2[class~="sidebar"]) are more specific and less flexible than class selectors. They can lead to unexpected results if not carefully managed. Class selectors are generally more predictable and should be preferred for styling unless you have a specific reason to use attribute selectors.